package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "deliveryapp",
```

This should align with our conformed terminology. See: https://github.com/ably/asset-tracking-backend-demo#delivery-lifecycle

Perhaps:

```suggestion
  "name": "Ably Asset Tracking Web Customer Demo",
```

---

Sorry, I just realised when working on something unrelated, that this name is not valid for use with `npm`.

We need to consider whether we should scope it (e.g. prefix with `@ably/`).

So, perhaps `@ably/asset-tracking-web-customer-demo`. What do you think, @owenpearson @Peter-Maguire @stmoreau?